Elevel/Elevel.Api/Controllers/TopicController.cs

```diff
@@ -0,0 +1,52 @@
+﻿using Elevel.Application.Features.TopicCommands;
+using MediatR;
+using Microsoft.AspNetCore.Mvc;
+using System;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Http;
+using Microsoft.Extensions.DependencyInjection;
+
+namespace Elevel.Api.Controllers
+{
+    [ApiController]
+    [Route("api/[controller]")]
+    public class TopicController : BaseApiController
+    {
+        private  IMediator _mediator;
+        protected IMediator Mediator => _mediator ??= HttpContext.RequestServices.GetService<IMediator>();
```

Comment:
this row has already existed in BaseApiController and it shows in warning, please pay attention to them